must-gather/collection-scripts/gather_versions

```diff
@@ -74,8 +74,31 @@ fi
 log_command "Velero client version" "velero version --client-only"
 log_deployment "Velero" "velero"
 
-# Storage classes
-log_command "Storage classes" "oc get storageclass --all-namespaces --insecure-skip-tls-verify=${skip_tls}"
+# Default storage classes
```

Comment:
fwiw.. it's a nice add to list the default storage classes, however if we have a dump of all the storage classes we're probably good. Perhaps others have a different opinion

Reply:
We also want volumesnapshotclasses yaml that has the CSI velero label per https://velero.io/docs/v1.11/csi/#implementation-choices

`  velero.io/csi-volumesnapshot-class: "true"
`

Reply:
Okay, both of these should be taken care of now.